### PR TITLE
Fix GPIO table overflow issue by updating div style for GPIO table

### LINKED ISSUE #536: Table Overflow Issue on ZERO Family Pages
The tables on the ZERO Family pages: 

- [ZERO](https://docs.radxa.com/en/zero/zero/hardware-design/hardware-interface)
- [ZERO 2 Pro](https://docs.radxa.com/en/zero/zero2pro/hardware-design/hardware-interface)
- [ZERO 3W/3E](https://docs.radxa.com/en/zero/zero3/hardware-design/hardware-interface)

 do not scroll horizontally due to the current `<div>` class structure.

### PR DESCRIPTION
**Pull request by Paschalis**
### Summary of Changes
This pull request addresses the issue of the GPIO table not scrolling horizontally on the hardware interface page. The change involves updating the `<div>` that wraps the GPIO table to include a style for overflow.

### Changes Made
- Updated the div from `<div className='gpio_style'>` to `<div className='gpio_style' style={{ overflow: "auto" }}>`.

Closes #536

### CODE REVIEW

**Comment by Paschalis** [comment]
Hi @peterwang2050, I’m having an issue with some failed checks after pushing recent changes to the scrollable-tables branch. I’ve ensured that the local code passed all necessary tests and reset the last two commits, but the checks still show as failed.. Could you provide some guidance on why this might still be failing, or if there’s a particular commit format or additional step I might have missed?

**Comment by RadxaYuntian** [comment]
There are extra empty line in your changes: https://github.com/radxa-docs/docs/actions/runs/11575838227/job/32223379144?pr=537#step:3:231

Please [install pre-commit](https://pre-commit.com/#installation) and run `pre-commit run --all-files` to fix styling issue locally.

File: i18n/en/docusaurus-plugin-content-docs/current/zero/zero/hardware-design/hardware-interface.md [hunk 1]
```diff
@@ -41,7 +41,6 @@ The ZERO provides a 40 pin GPIO header that is compatible with most peer accesso
 
 **Hint: Actual compatibility is based on usage. **
 
-<div className='gpio_style'>
```
Comment (RadxaYuntian) [inline i18n/en/docusaurus-plugin-content-docs/current/zero/zero/hardware-design/hardware-interface.md:44]: If you can't see the action log, then it is the change like this. You now have 2 consecutive empty lines.

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

**Comment by Paschalis** [comment]
Thank you, @RadxaYuntian, for your guidance!

File: i18n/en/docusaurus-plugin-content-docs/current/zero/zero/hardware-design/hardware-interface.md [hunk 2]
```diff
@@ -41,15 +41,15 @@ The ZERO provides a 40 pin GPIO header that is compatible with most peer accesso
...
-
 :::caution
 Depending on the hardware version, one of the pins 35 and 38 is connected to the power LED instead of the 40-pin header. If your design uses these pins, please check the hardware version before purchasing.  
 GPIOAO_8 and GPIOAO_10 are available in hardware versions v1.51 and higher.  
 GPIOA_14 and GPIOA_15 are connected to pull-up resistors and USB-C controllers and therefore cannot be used for general GPIOs.  
 Pin 22 (GPIOC_7) and pin 36 (GPIOH_8) are open-drain pins. This means that they need to be connected to GND or VCC (floating state undefined), and they need an external pull-up when used as inputs. In addition, GPIOH_8 utilizes a 5V logic level.
 :::
 
+<div className='gpio_style' style={{ overflow :"auto"}} >
```
Comment (RadxaYuntian) [inline i18n/en/docusaurus-plugin-content-docs/current/zero/zero/hardware-design/hardware-interface.md:51]:
```suggestion
<div className='gpio_style' style={{ overflow :"auto" }}>
```
Could use an extra space to match.

File: i18n/en/docusaurus-plugin-content-docs/current/zero/zero2pro/hardware-design/hardware-interface.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ The ZERO 2 PRO provides a 40 pin GPIO socket that is compatible with most SBC ac
 
 **Hint: Actual compatibility is subject to usage.**
 
-<div className='gpio_style'>
+<div className='gpio_style' style={{ overflow :"auto"}} >
```
Comment (RadxaYuntian) [inline i18n/en/docusaurus-plugin-content-docs/current/zero/zero2pro/hardware-design/hardware-interface.md:36]:
```suggestion
<div className='gpio_style' style={{ overflow :"auto" }}>
```

File: i18n/en/docusaurus-plugin-content-docs/current/zero/zero3/hardware-design/hardware-interface.md [hunk 4]
```diff
@@ -33,12 +33,12 @@ Radxa ZERO 3 provides a 40-pin GPIO header, which is compatible with most SBC ac
 
 **Tips: Actual compatibility is based on usage **
 
-<div className='gpio_style'>
-
 :::tip
 Pin 3, Pin 5, Pin 27, and Pin 28 add extra pull-up resistors for I2C device power supply, so they work abnormally when used as GPIOs.
 :::
 
+<div className='gpio_style' style={{ overflow :"auto"}} >
```
Comment (RadxaYuntian) [inline i18n/en/docusaurus-plugin-content-docs/current/zero/zero3/hardware-design/hardware-interface.md:40]:
```suggestion
<div className='gpio_style' style={{ overflow :"auto" }}>
```

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
Please also split the other pre-commit fixes to its own commit, and the GPIO table fix in another commit.

**Review by peterwang2050 — APPROVED** [review]

**Review by RadxaYuntian — APPROVED** [review]

**Review by peterwang2050 — APPROVED** [review]

**Pull request merged**